core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -131,6 +136,7 @@ public enum SqlConformanceEnum implements SqlConformance {
     case MYSQL_5:
     case PRESTO:
     case SNOWFLAKE:
+    case SPARK:
```

Comment:
Can you please check if handling for SPARK is required for below functions as well : isHavingAlias(), semantics(), allowNiladicParentheses()

Reply:
Not in Scope..

Reply:
RTP-208 created for out of scope functions.